src/shared/store/ApiStore/ApiStore.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiResponse, IApiStore, RequestParams } from './types';
+import qs from 'qs';
+
+export default class ApiStore implements IApiStore {
+    readonly baseUrl: string = 'https://api.github.com/';
```

Comment:
`ApiStore` - абстрактный стор, он не должен внутри знать  что может работать с api.github, ему неважна апишка
```suggestion
    readonly baseUrl: string;
```